internal/utils/moduleversionlabels.go

```diff
@@ -8,15 +8,22 @@ import (
 )
 
 func GetModuleVersionLabelName(namespace, name string) string {
-	return fmt.Sprintf("%s.%s.%s", constants.ModuleVersionLabelPrefix, namespace, name)
+	return getVersionLabelName(constants.ModuleVersionLabelPrefix, namespace, name)
 }
 
 func GetModuleLoaderVersionLabelName(namespace, name string) string {
-	return fmt.Sprintf("%s.%s.%s", constants.ModuleLoaderVersionLabelPrefix, namespace, name)
+	return getVersionLabelName(constants.ModuleLoaderVersionLabelPrefix, namespace, name)
 }
 
 func GetDevicePluginVersionLabelName(namespace, name string) string {
-	return fmt.Sprintf("%s.%s.%s", constants.DevicePluginVersionLabelPrefix, namespace, name)
+	return getVersionLabelName(constants.DevicePluginVersionLabelPrefix, namespace, name)
+}
+
+func getVersionLabelName(labelPrefix, namespace, name string) string {
+	if namespace == "" {
+		namespace = "default"
+	}
+	return fmt.Sprintf("%s.%s.%s", labelPrefix, namespace, name)
```

Comment:
WDYT about:
```suggestion
	return fmt.Sprintf("%s/%s.%s.%s", prefix, namespace, name, property)
```
Where `prefix` is `kmm.node.kubernetes.io`.
This groups all labels for a given module together then they are sorted alphabetically.
Currently we set `{prefix}/{moduleName}.{ready}`.

Reply:
Not sure i understand the change you suggests: this module deals with module version labels. So, what do we get by setting the prefix explicitly? Currently it is part of the LabelPrefix defined in the constants.go.

Reply:
I meant that I would have the property (e.g. `version-module`) after the `namespace.name` in the label.